internal/operator-controller/rukpak/convert/registryv1.go

```diff
@@ -29,6 +29,10 @@ import (
 	"github.com/operator-framework/operator-controller/internal/operator-controller/rukpak/util"
 )
 
+const (
+	AnnotationRegistryV1GeneratedManifest = "io.operatorframework.olm.generated-manifest"
```

Comment:
So it is a new annotation
Could you please help me understand ( low verbose overview ) 
So, if we add this I am as an Cluster extension User will be able to ...

Reply:
This is something we can talk about. This annotation exists only to inform the client of `convert.Convert` which of the resources where generated by it as opposed to coming directly from the bundle. Other than that, it does nothing. It's really just a signal to say "this resource was generated when rendering the bundle - it is not part of the bundle itself"